web/src/main/java/com/bazaarvoice/emodb/web/scanner/writer/S3ScanWriter.java

```diff
@@ -123,8 +128,9 @@ public void progressChanged(ProgressEvent progressEvent) {
                             }
                         };
 
-                        PutObjectRequest putObjectRequest = new PutObjectRequest(_bucket, _key, _file);
-                        putObjectRequest.setGeneralProgressListener(progressListener);
+                        PutObjectRequest putObjectRequest = new PutObjectRequest(_bucket, _key, _file)
+                                .withTagging(new ObjectTagging(ImmutableList.of(new Tag("bv.emodb.table", _tableName))))
+                                .withGeneralProgressListener(progressListener);
```

Comment:
This one addition of `withTagging()` to the `PutObjectRequest` is the actual crux of this PR.  Most of the rest just supports the SDK upgrade.